openedx/features/qverse_features/registration/validators.py

```diff
@@ -23,21 +23,28 @@ def validate_admission_file(file):
                 'regno', 'firstname', 'surname', 'othername', 'levelid',
                 'programmeid', 'departmentid', 'mobile', 'email'
                 ]
-    header_row = []
+
     try:
         file_content = file.read()
         
         try:
-            header_row = get_file_header_row(file_content, 'utf-8')
+            file_rows = get_file_rows(file_content, 'utf-8')
         except Error:
-            header_row = get_file_header_row(file_content, 'utf-16')
-
+            file_rows = get_file_rows(file_content, 'utf-16')
+    except TypeError:
+        raise ValidationError('Invalid file content. Csv should only use "," as delimiter')
     except Exception:
         raise ValidationError('Invalid file encoding format. Only utf-8 and utf-16 file encoding formats are supported.')
 
+    header_row = file_rows[0]
```

Comment:
what if file_rows is an empty list? Is it handled?

Reply:
It never will be. An exception will be raised before it happens.